api.md

```diff
@@ -689,6 +687,14 @@ If space partitioning is not being used, the `chunk_time_interval`
 should be the same as the non-distributed case, as all of the incoming
 data will be handled by a single node.
 
+**Replication factor:** `relication_factor` defines to how many data nodes
+newly created chunk will be replicated. So if a data node is removed, no data
+will be lost, but the chunks, which were present on the removed data node, will
+be less replicated, underreplicated. Currently it is not possible to get back the 
+replication of the underreplicated chunks. While newly created chunks will be created
+with the required number of replicas. To avoid such inconsistency we don't recommend
+to use `replication_factor` bigger than 1.
```

Comment:
```suggestion
**Replication factor:**  The hypertable's `replication_factor` defines to how
many data nodes a newly created chunk will be replicated.  That is, a chunk
with a `replication_factor` of three will exist on three separate data nodes,
and rows written to that chunk will be inserted (as part of a two-phase
commit protocol) to all three chunk copies.  For chunks replicated more
than once, if a data node fails or is removed, no data will be lost, and writes
can continue to succeed on the remaining chunk copies.  However, the chunks
present on the lost data node will now be under-replicated than the specified
replication factor.  Currently, it is not possible to restore under-replicated
chunks, although this limitation will be removed in a future release. To avoid
such inconsistency, we do not yet recommend using `replication_factor` > 1,
and instead rely on physical replication of each data node.
```